File: minwook/7주차/행렬의_덧셈.java
class Solution {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int[][] answer = new int[arr1.length][arr1[0].length];
        for(int i=0; i<arr1.length; i++){
            for(int j=0; j<arr1[0].length; j++){
                answer[i][j] = arr1[i][j] + arr2[i][j]; // 2중 반복문으로 행렬에 접근 후 덧셈
            }
        }
        return answer;
    }
}

File: yongkeun/7week/행렬의덧셈.js
function solution(arr1, arr2) {
  var answer = [];
  for (let i = 0; i < arr1.length; i++) {
    answer[i] = []; //i번째 방의 배열을 초기화
    for (let j = 0; j < arr1[i].length; j++) {
      answer[i][j] = arr1[i][j] + arr2[i][j]; // arr1의 00과 arr2의 00을 더하여 answer00에 넣기
    }
  }
  return answer;
}
